backend/api_calls.py

```diff
@@ -1055,6 +1056,29 @@ def MarkNotificationSeen(request, firebase):
       request['playerId'], request['notificationId']), put_data)
 
 
+def RegisterUserDevice(request, firebase):
+  """Register a user device to a userId.
+
+  Validation:
+    userId must exist.
+
+  Args:
+    userId: User id to associate with.
+    deviceToken: Ionic device token.
```

Comment:
Why would we not have a method that updates user data? I mean, that's exactly what this method is doing, right? This method is essentially an UpdateUser that only supports on entry in the user data.